e2e/paths/enterRespondents_test.js

```diff
@@ -1,27 +1,27 @@
 const config = require('../config.js');
 const respondent = require('../fixtures/respondent.js');
 
-Feature('Enter respondents').retry(2);
+Feature('Enter respondents');
```

Comment:
are we not keeping retry on our tests?

Reply:
Yes sorry, had removed for testing purposes. Will put back in.

Reply:
Good spot @swalker125